sgrep_lint/sgrep-lint

```diff
@@ -3,4 +3,4 @@
 # sgrep-lint single executable must be run from the folder with all the shared objects
 
 THIS_DIR="$(dirname "$(realpath "$0")")";
-LD_PRELOAD="/bin/sgrep-lint-files:/usr/local/bin/sgrep-lint-files" ${THIS_DIR}/sgrep-lint-exe "$@"
```

Comment:
Should verify that this does not break OSX Builds.

Reply:
Tried locally setting up sgrep-lint and it looks like everything is ok?

Reply:
Confirmed this branch works with make && make install && sgrep-lint --help

Nice!